### PR TITLE
Packaging

### PR DESCRIPTION
**Pull request by suryatejreddy**

### CODE REVIEW

File: setup.cfg [hunk 1]
```diff
@@ -0,0 +1,15 @@
+[metadata]
+name = evatestdb
+version = 0.0.3
+author = Georgia Tech Database Group
+author_email = svyalla3@gatech.edu
```
Comment (gaurav274) [inline setup.cfg:5]: Use the DB email id Joy shared

File: test/eva_cmd_client.py [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+import mock
+from eva.eva_cmd_client import main
+
+
+class CMDClientTest(unittest.TestCase):
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+
+    @mock.patch('eva.eva_cmd_client.eva_client')
+    def dummyTest(self):
```
Comment (gaurav274) [inline test/eva_cmd_client.py]: rename it to start with **test_**

File: test/eva_server.py [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+from eva.eva_server import main
+
+
+class EVAServerTest(unittest.TestCase):
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+
+    @mock.patch('eva.eva_server.eva')
+    @mock.patch('eva.udfs.udf_bootstrap_queries.init_builtin_udfs')
+    def dummyTest(self):
```
Comment (gaurav274) [inline test/eva_server.py]: rename to start with **test_**

**Review by gaurav274 — CHANGES_REQUESTED** [review]

**Comment by xzdandy** [comment]
A dump question: how shall I test this packaging on a clean machine? Do we still need the installation scripts (script/install) and mysql configuration first?

**Comment by suryatejreddy** [comment]
> A dump question: how shall I test this packaging on a clean machine? Do we still need the installation scripts (script/install) and mysql configuration first?



> A dump question: how shall I test this packaging on a clean machine? Do we still need the installation scripts (script/install) and mysql configuration first?

@xzdandy You can install it using this sample pypi package https://pypi.org/project/evatestdb/. 

use `eva_server` and `eva_client` to start the server and client. Let me know if it doesn't work on your machine.

**Comment by xzdandy** [comment]
> @xzdandy You can install it using this sample pypi package https://pypi.org/project/evatestdb/.
> 
> use `eva_server` and `eva_client` to start the server and client. Let me know if it doesn't work on your machine.

@suryatejreddy Thanks, I will try it. PS: Is `pip install evatestdb` same as `python setup.py install` in this pr (branch suryatejreddy:packaging)?

**Comment by gaurav274** [comment]
Duplicate

**Pull request closed without merging**